ansible/roles/openshift_outbound_tcp_logging/tasks/main.yml

```diff
@@ -0,0 +1,121 @@
+---
+- set_fact:
+    outbound_logging_chain:
+    - chain: oo-outbound-logging
+      line:  ":oo-outbound-logging - [0:0]"
+      after: "filter$"
+      regex: "Create oo-outbound-logging chain"
+      test: True
+      pos: 1
+
+    outbound_logging_rules:
+    - chain: oo-outbound-logging
+      line:  "-A oo-outbound-logging -m comment --comment \"Log outbound TCP connections with this chain\" -m limit --limit 50/second --limit-burst 50 -j LOG --log-prefix \"tcp_out_logging: \""
```

Comment:
I don't think we need implementation comments in the iptables rules themselves. Since you do need an identifier to match your regexes on, I think you could just create an identifier for each rule that will for sure be unique. I'd rather see something like:
```
outbound_logging_rules:
# Log new outbound TCP connections (capped at 50/second)
- chain: oo-outbound-logging
  line:  "-A oo-outbound-logging -m comment --comment id_outbound_tcp_log_1_ -m limit --limit 50/second --limit-burst 50 -j LOG --log-prefix \"tcp_out_logging: \""
  after: "^-A FORWARD"
  ...
```